runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -773,6 +773,28 @@ static TR::Node *getStoreValueBaseNode(TR::Node *storeValueNode, TR::SymbolRefer
    return storeValueBaseNode;
    }
 
+static void populatePrexArgInfo(OMR::ValuePropagation *vp, TR_PrexArgInfo *argInfo, TR::Node *node, bool isGlobal, bool trace)
```

Comment:
It seems redundant to take `trace` as a parameter when you could get the same information from `vp->trace()`